### ISSUE TITLE
net/krb5 - verto.h missing

### ISSUE THREAD

**Issue by db260179** [openwrt/packages#1143]
Hi,

When compiling, you get 'include verto.h is missing' error.

Please add the the following to Makefile to fix this issue

$(CP) $(PKG_INSTALL_DIR)/usr/include \
                $(1)/usr/include

So it looks like
define Build/InstallDev
        $(INSTALL_DIR) $(1)/usr/include
        $(CP) $(PKG_INSTALL_DIR)/usr/include \
                $(1)/usr/include/krb5
        $(CP) $(PKG_INSTALL_DIR)/usr/include \
                $(1)/usr/include
        $(INSTALL_DIR) $(1)/usr
        $(CP) $(PKG_INSTALL_DIR)/usr/lib \
                $(1)/usr
        rm -f $(1)/usr/lib/libcom_err*
endef

Thanks

**Comment by plntyk** [comment]
Do you mean that error:
```
In file included from net-server.c:60:0:
../../include/net-server.h:32:19: fatal error: verto.h: No such file or directory
 #include <verto.h>
```

I only hit this error if I do a parallel build (make -j 8) - is that your case too ?

**Comment by db260179** [comment]
compiling with just - make V=s

Produces this problem

**Comment by psycho-nico** [comment]
Fixed by f746041323d9e35cd311c9280c06994baae017c4

Thanks !
